docs/data/routes/docs/techniques/extending-layout-service/extending-layout-service-overview/en.md

```diff
@@ -141,18 +140,40 @@ changes to your configuration, should the `jss` configuration change during an u
 </layoutService>
 ```
 
+After defining your custom configuration, you may also wish to add the configuration name to the list of `<AllowedConfigurations>` for the `renderJsonRendering` pipeline. Doing so will ensure that the `PlaceholderTransformer` code that executes for the default `jss` configuration will execute for your custom configuration as well.
+
+> Note: This is only relevant if your custom configuration will be delivering output that is similar in shape to the default JSS configuration. If your custom configuration is not related to JSS or doesn't depend on anything from the default `jss` configuration, this step is not necessary.
+
+```xml
+<pipelines>
+  <group groupName="layoutService">
+    <pipelines>
+      <renderJsonRendering>
+        <processor type="Sitecore.JavaScriptServices.ViewEngine.LayoutService.Pipelines.RenderJsonRendering.AddComponentName, Sitecore.JavaScriptServices.ViewEngine" resolve="true">
```

Comment:
Can this be delivered as a copy n' paste-able config patch file?

Reply:
It can, and it will.